1/ex1.py

```diff
@@ -0,0 +1,8 @@
+print "Hello World!"
+#print "Hello Again"
```

Comment:
Try highlighting all of these lines in Sublime and hit `Cmd + /`. Try it again, what did it do?